lib/pharos/phases/migrate_master.rb

```diff
@@ -6,19 +6,20 @@ class MigrateMaster < Pharos::Phase
       title "Migrate master"
 
       def call
-        migrate_0_5_to_0_6 if migrate_0_5_to_0_6?
```

Comment:
Removing old migration because we don't really support direct upgrade from 0.5.x to 1.2.x.